spec/koans.js

```diff
@@ -84,38 +85,44 @@ describe('`const` is like `let` plus read-only. ', () => {
 describe('`string.includes()` finds string within another string. ', () => {
 
   describe('find a single character', function() {
-    it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+    it('in a three char string', () => {
+      const searchString = ('xyz')
+      expect('xyz'.includes(searchString)).toBe(true);
     });
     it('reports false if character was not found', function() {
-      /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+      const expected = false
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
   describe('find a string', function() {
     it('that matches exactly', function() {
-      /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+      const findSome = (i) => 'xyz'.includes(i);
+
+      // function findSome (par) {
+      //   return "xyz".includes(par);
+      // }
+
+      expect(findSome('xyz')).toBe(true);
     });
   });
 
   describe('search for an empty string, is always true', function() {
     it('in an empty string', function() {
-      /* .... */
-      //expect(''.includes(x)).toBe(true);
+      const x = ''
+      expect(''.includes(x)).toBe(true);
     });
     it('in `abc`', function() {
-      /* .... */
-      //expect('abc'.includes(x)).toBe(true);
+      let x = 'abc';
+      expect('abc'.includes(x)).toBe(true);
     });
   });
 
   describe('takes a position from where to start searching', function() {
     it('does not find `a` after position 1 in `abc`', function() {
-      /*....*/
-      //expect('abc'.includes('a', position)).toBe(false);
+      const str = 'abc';
+      return str.startsWith(1);
+      expect('abc'.includes('a', 1)).toBe(false);
```

Comment:
There is a small problem here, if you put a return statement before the 'expect', the expect code (in line 125) can never be executed. 
The original code was : 
```js
 it('does not find `a` after position 1 in `abc`', function() {
      /*....*/
      //expect('abc'.includes('a', position)).toBe(false);
}
```
To make the test runnable, you can just decalare a position variable set to 1 : 
```js
 it('does not find `a` after position 1 in `abc`', function() {
      const position = 1
      expect('abc'.includes('a', position)).toBe(false);
}
```